drivers/media/i2c/ov5693.c

```diff
@@ -910,6 +916,16 @@ static int ov5693_s_ctrl(struct v4l2_ctrl *ctrl)
 	struct i2c_client *client = v4l2_get_subdevdata(&dev->sd);
 	int ret = 0;
 
+	/* If VBLANK is altered we need to update exposure to compensate */
+	if (ctrl->id == V4L2_CID_VBLANK) {
+		int exposure_max;
+		exposure_max = dev->mode->lines_per_frame - 8; // set 123 for sensible display
```

Comment:
This comment looks like a leftover so remove it.

Reply:
Whoops! That was me diagnosing why my image was nuclear explosion white all of a sudden. Thanks, I'll take them out.

Reply:
Removed both these - thanks.